srcmedia/scss/cards/_result.scss

```diff
@@ -70,5 +83,20 @@ ol.card-results {
         color: $mid-grey;
     }
 
+
+    // adjust for individual member card thumbnails
+    .member-cards & {
+        margin-bottom: 1.75rem;
+
+        .card-dates {
+            // NOTE: picture element is taking up some space so this
+            // looks too big; not sure how to adjust
+            // padding-top: 0.95rem;
```

Comment:
is this still an issue?

Reply:
Yes - at least, the way I'm looking at the elements in Chrome that's how it looks to me. I'm not sure how to adjust or if I'm misunderstanding what's going on, but when I add that padding it seems like too much.